test/extensions/transport_sockets/tls/ocsp/ocsp_test.cc

```diff
@@ -189,16 +189,28 @@ TEST_F(Asn1OcspUtilityTest, ParseMethodWrongTagTest) {
   expectThrowOnWrongTag(Asn1OcspUtility::parseResponseStatus);
 }
 
-TEST_F(Asn1OcspUtilityTest, ParseResponseDataBadResponderIdVariantTest) {
+TEST_F(Asn1OcspUtilityTest, ParseResponseDataUnsupportedVersionTest) {
   std::vector<uint8_t> data = {
       // SEQUENCE
-      0x30,
-      7,
-      // version
+      0x30, 5,
+      // invalid version field, the value of v2 is 1
       0xa0,
-      0x02,
+      0x03, // bit string
```

Comment:
Where'd this come from? This isn't a BIT STRING tag. It's just a length. Rather than try to hand-craft inputs like this, Just use https://github.com/google/der-ascii and including the structure in the comment:

```
// SEQUENCE {
//   # Invalid version field. The value of v1 is 0, not 1. 
//   [0] { INTEGER { 1 } }
// }
std::vector<uint8_t> data = {0x30, 0x05, 0xa0, 0x03, 0x02, 0x01, 0x01};
```

Reply:
It is so cool, I will update.

Reply:
Ah, I obviously misunderstood something in the let's encrypt ASN.1 primer. Thanks for setting us straight again, @davidben.